packages/components/src/DataGridPagination/DataGridPagination.tsx

```diff
@@ -18,13 +26,23 @@ export type DataGridPaginationProps = Omit<PaginationProps, 'count'> & {
    * Текущая страница
    */
   page: number;
+  /**
+   * Выбор количества записей на странице с помощью селектора
+   */
+  onSetCount?: (rowsPerPage: number) => void;
+  /**
+   * Массив значений селектора
+   */
+  selectOptions?: number[];
 };
 
 export const DataGridPagination = ({
   page,
-  rowsPerPage = 10,
+  rowsPerPage = 20,
   totalCount,
   className,
+  onSetCount,
```

Comment:
```suggestion
  onSetCountPerPage,
```

думаю будет более явно говорить о чем речь